### ISSUE TITLE
Editorial - change "this/current version" to specific version & date

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#976]
As per discussion in TF meeting of 
https://www.w3.org/2026/07/30-wcag2ict-minutes.html

There are a couple of places in the text we we have "latest" or "current". 
We should change to WCAG 2.2. + erratas as of mm/dd/yyyy

**Comment by pday1** [comment]
https://w3c.github.io/wcag2ict/#background
Para 1. Sentence 2. 

> The current version also includes Level AAA success criteria.

https://w3c.github.io/wcag2ict/#interpretation-of-web-terminology-in-a-non-web-context
Last para, last sentence

> Level AAA success criteria were not addressed by previous versions of the WCAG2ICT Note but are now included in this version.

**Comment by pday1** [comment]
Proposal was to specify version and date

> WCAG2ICT 2.2 (dd Month yyyy)

However, we cannot do that until this new version of WCAG2ICT (containing AAA guidance) is published.


Placeholder text is therefore shown below with deletions in ~~strikethrough~~ and new content in **bold**:

https://w3c.github.io/wcag2ict/#background
Para 1. Sentence 2. 

> ~~The current version~~ **WCAG2ICT version 2.2 (DD Month YYYY)**  also includes Level AAA success criteria.

https://w3c.github.io/wcag2ict/#interpretation-of-web-terminology-in-a-non-web-context
Last para, last sentence

> Level AAA success criteria were not addressed by previous versions of the WCAG2ICT Note but are now included in ~~this version~~ **WCAG2ICT version 2.2 (DD Month YYYY)**.

**Comment by bruce-usab** [comment]
I know the URL for WCAG2ICT is https://www.w3.org/TR/wcag2ict-22/ -- but that reflects addressing WCAG 2.2, not that this is version 2.2 of WCAG2ICT.   It also implies that there is a WCAG2ICT version 2.1.   Citing the DD Month YYYY of publication is fine.  I recommend resisting the temptation to refer to WCAG2ICT by decimal number versioning.
